src/App.tsx

```diff
@@ -21,6 +40,8 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={perPage}
+            onChange={(e) => handleOnChange(e)}
```

Comment:
```suggestion
            onChange={handleOnChange}
```
this can be shortened